sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -295,6 +302,7 @@ public TableType getJdbcTableType()
                   val.getValue().isPublished(),
                   val.getValue().isAvailable(),
                   val.getValue().isRealtime(),
+                  AVAILABLE_IS_OVERSHADOWED_VALUE,
```

Comment:
i think this should have a comment that this assumes unpublished segments are never overshadowed.

Reply:
added comment